src/server.py

```diff
@@ -0,0 +1,93 @@
+import mysql.connector
+
+from json import dumps
+
+from sanic import Sanic
+from sanic.response import json
+
+
+app = Sanic()
+
+mydb = mysql.connector.connect(
+    host='db',
+    user='root',
+    passwd='password',
+    database='cat_db'
+)
+mycursor = mydb.cursor()
+
+
+@app.route('/get_db', methods=['GET'])
+async def get_db(request):
+    try:
+        mycursor.execute('SHOW DATABASES')
```

Comment:
Also, can you please use an SQL query builder (like [pypika](https://pypika.readthedocs.io/en/latest/) nebo [SQLAlchemy Core](https://docs.sqlalchemy.org/en/14/core/tutorial.html)) or is this something you plan to do later?